packages/shortest/src/browser/core/browser-tool.ts

```diff
@@ -440,7 +513,37 @@ export class BrowserTool extends BaseBrowserTool {
           }
         }
 
-        case "sleep": {
+        case InternalActionEnum.WAIT:
+          if (!input.duration) {
+            throw new ToolError("Duration required for wait action");
+          }
+          const seconds = input.duration;
+          await this.page.waitForTimeout(seconds * 1000);
+          output = `Waited for ${seconds} second${seconds !== 1 ? "s" : ""}`;
+          break;
+
+        case InternalActionEnum.SCROLL:
```

Comment:
This is _very_ slow. `scroll_amount` is measured in _clicks_ (possible pixels). During some tests (e.g. _scroll to the bottom of page_), the value returned by AI was 10-20. Hopefully they improve the functionality over time.